build.sbt

```diff
@@ -205,9 +205,9 @@ lazy val core = crossProject(JSPlatform, JVMPlatform)
     name := "enumeratum",
     version := Versions.Core.head,
     crossScalaVersions := scalaVersionsAll,
-    libraryDependencies += "com.beachape" %% "enumeratum-macros" % Versions.Macros.stable
+    //libraryDependencies += "com.beachape" %% "enumeratum-macros" % Versions.Macros.stable
   )
-//  .dependsOn(macros) // used for testing macros
+  .dependsOn(macros) // used for testing macros
```

Comment:
Not sure what to do with this. If I put it back to how it was, the tests won't compile until a new version of the macros is published.